src/js/components/uploadDetachedFiles/UploadDetachedFileMeta.jsx

```diff
@@ -157,6 +158,9 @@ export default class UploadDetachedFileMeta extends React.Component {
 
 	checkFileStatus(submissionID) {
 		// callback to check file status
+		if(!submissionID){
```

Comment:
Sorry, got desperate to fix errors and at one point this was receiving submissionID = null. No longer seems to be the case